packages/cxl-ui/src/components/cxl-playbook-accordion.js

```diff
@@ -0,0 +1,156 @@
+import { customElement } from 'lit-element';
+import '@conversionxl/cxl-lumo-styles';
+import '@vaadin/vaadin-accordion';
+import '@vaadin/vaadin-checkbox';
+import { registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlPlaybookAccordionGlobalStyles from '../styles/global/cxl-playbook-accordion-css.js';
+import { CXLVaadinAccordion } from './cxl-vaadin-accordion';
+import vaadinAccordionPanelStyles from '../styles/cxl-playbook-accordion/vaadin-accordion-panel-css.js';
+import vaadinCheckboxStyles from '../styles/cxl-playbook-accordion/vaadin-checkbox-css.js';
+
+/**
+ * Allows opening multiple panels simultaneously.
+ * Saves panel state, restores on page load.
+ */
+@customElement('cxl-playbook-accordion')
+export class CXLPlaybookAccordion extends CXLVaadinAccordion {
+  /**
+   *
+   * @return {NodeListOf<Element>}
+   */
+  get accordionPanels() {
+    return this.querySelectorAll('vaadin-accordion-panel');
+  }
+
+  /**
+   *
+   * @return {NodeListOf<Element>}
+   */
+  get panelSummarySlotCheckboxes() {
+    return Array.from(this.accordionPanels).map((el) =>
```

Comment:
Upstream provides `_filterItems()` method for panels

https://github.com/vaadin/vaadin-accordion/blob/v2.0.0-alpha5/src/vaadin-accordion.js#L159-L166

Reply:
> Upstream provides `_filterItems()` method for panels
> 
> https://github.com/vaadin/vaadin-accordion/blob/v2.0.0-alpha5/src/vaadin-accordion.js#L159-L166

Related to:
https://github.com/conversionxl/aybolit/pull/87#discussion_r580863133